src/XMLexport.cpp

```diff
@@ -391,6 +391,7 @@ void XMLexport::writeHost(Host* pHost, pugi::xml_node mudletPackage)
     host.append_attribute("mRequiredDiscordUserName") = pHost->mRequiredDiscordUserName.toUtf8().constData();
     host.append_attribute("mRequiredDiscordUserDiscriminator") = pHost->mRequiredDiscordUserDiscriminator.toUtf8().constData();
     host.append_attribute("SemiColonOnlySGRColorCodesHaveColorSpaceId") = pHost->getHaveColorSpaceId() ? "yes" : "no";
+    host.append_attribute("ServerMayRedefineColors") = pHost->getMayRedefineColors() ? "yes" : "no";
```

Comment:
Could we still do the `m` prefix tradition? :pray:

Reply:
That is not helpful IMHO - that string is what appears in the XML file so should I think be comprehensible outside of the application itself - sadly *past* copying of the variable name even when it is `mNOT_THAT_GOOD` {er, `mUSE_FORCE_LF_AFTER_PROMPT`} does not seem helpful to me and it is difficult to clean up past cases. Better I think to have a long, but readable text - unless it is an element that appears very frequently - so that it documents itself (as putting explanatory comments *in there* is a whole different matter that in a C header file)... :slightly_smiling_face: